lib/rex/post/smb/ui/console/command_dispatcher/shares.rb

```diff
@@ -53,7 +53,7 @@ def initialize(console)
           #
           def commands
             cmds = {
-              'shares' => 'View the available shares and interact with one',
+              'shares' => 'View the available shares and interact with one. Use the -h flag for more info',
```

Comment:
```suggestion
              'shares' => 'View the available shares and interact with one',
```

I think adding this specifically to the commands maybe isn't the right place. What would you think about maybe looking for a way to add this to the top level command. That way all help commands would benefit from having some extra affordance.

My thinking would be in terms off output it would be similar to how the search command has an output at the end of the command that with give some information on how to interact with modules, it will even take a module from the list and add an example for how to interact with that module.
![image](https://github.com/rapid7/metasploit-framework/assets/69522014/165b283c-4030-4504-9712-39dfac9ef6d6)

I'm thinking the ideal solution may be something similar, where the help output you added specifically to the command could be instead output at the bottom, and also give examples like the search commands output. By also taking a random command from the available commands for the command dispatchers available at that time would allow you take a random command and have it be relevant to each help command.

The green highlighting would also increase the chances of users seeing the output.

Something like:
```
Get command specific help by running `<command> -h` or `<command> help`. For example `shares -h` or `shares help` 
```

Reply:
* Note instead of just using backticks, we'd obviously use a green highlight color